recipe/meta.yaml

```diff
@@ -14,11 +14,13 @@ source:
   - fix-compile.patch
 
 build:
-  number: 4
+  number: 5
   skip: True  # [py2k]
   script:
-    - export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
-    - export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -I${PYTHON_INCLUDE_DIR}"
+    - export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"                                            # [unix]
+    - export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -I${PYTHON_INCLUDE_DIR}"              # [unix]
+    # see https://github.com/boostorg/system/issues/32#issuecomment-462912013
+    - set "CXXFLAGS=%CXXFLAGS% /DHAVE_SNPRINTF=1"                                            # [win]
```

Comment:
Not sure why this isn't working. That symbol fixed equivalent failures elsewhere. Since I don't see it in the logs, I'm kinda suspecting it somehow isn't passed on to the setup invocation.